src/main/java/com/faforever/client/game/GameRangePredicate.java

```diff
@@ -0,0 +1,33 @@
+package com.faforever.client.game;
+
+import com.faforever.client.player.Player;
+import com.faforever.client.util.RatingUtil;
+
+import java.util.Optional;
+import java.util.function.Predicate;
+
+public class GameRangePredicate implements Predicate<Game> {
```

Comment:
this is already done by the server

Reply:
Makes sense... But also that makes me a sad panda.